kubernetes/apps/charts/postgresql-backup/templates/cronjob.yaml

```diff
@@ -53,10 +57,29 @@ spec:
                       --stdin \
                       --stdin-filename pg_dumpall.sql.gz
 
+                  sql_snapshot_status=$?
+
                   # prune aged snapshots
                   echo "Pruning aged snapshots"
                   restic forget \
                     --host {{ .Release.Namespace }}:{{ .Release.Name }} \
                     --keep-last 36 \
                     --keep-daily 7 \
                     --keep-weekly 52
+
+                  prune_status=$?
+
+                {{ if .Values.healthchecks.uuid }}
+                  backup_status=0
+
+                  if [ $sql_snapshot_status -ne 0 ]; then
+                    echo "Reporting failure: SQL snapshot"
+                    backup_status=1
+                  elif [ $prune_status -ne 0 ]; then
+                    echo "Reporting failure: Prune"
```

Comment:
I personally read this error message as communicating a failure to report the status to the health checker. I would recommend an error message which reads something more like "Backup error: [restic snapshot|restic forget|etc.]"

Reply:
Can do, I just took this from some other random live cluster app